### PR TITLE
Sprint 21 TLT-1325 Adding has_lti_role utility function

### PR DESCRIPTION
**Pull request by douglashall**
@rascalking

### CODE REVIEW

File: django_auth_lti/verification.py [hunk 1]
```diff
@@ -14,9 +15,14 @@ def is_allowed(request, allowed_roles, raise_exception):
...
+    is_user_allowed = set(allowed) & set(user_roles)
     
     if not is_user_allowed and raise_exception:
         raise PermissionDenied
     
     return is_user_allowed
+
+
+def has_lti_role(request, role):
+    roles = request.session['LTI_LAUNCH'].get('roles', [])
```
Comment (sapnamysore) [inline django_auth_lti/verification.py]: Does it make sense to check and raise ImproperlyConfigured exception here as well?

**Comment by sapnamysore** [comment]
Looks good :+1:

File: django_auth_lti/verification.py (hunk 1 above)
Comment (douglashall) [inline django_auth_lti/verification.py]: Will do.

**Comment by rascalking** [comment]
:+1:

**Pull request merged**